target_chains/ton/contracts/tests/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Cell, beginCell } from "@ton/core";
+
+export function createCellChain(buffer: Buffer): Cell {
+  let chunks = bufferToChunks(buffer, 127);
+  let lastCell: Cell | null = null;
+  // Iterate through chunks in reverse order
+  for (let i = chunks.length - 1; i >= 0; i--) {
+    const chunk = chunks[i];
+    const cellBuilder = beginCell();
+    const buffer = Buffer.from(chunk);
+    cellBuilder.storeBuffer(buffer);
+
+    if (lastCell) {
+      cellBuilder.storeRef(lastCell);
+    }
+
+    lastCell = cellBuilder.endCell();
+  }
+
+  // lastCell will be the root cell of our chain
+  const rootCell = lastCell!;
```

Comment:
nit: probably better to throw an error if it's undefined as this behaviour can happen with an empty buffer as input